source/release-notes.txt

```diff
@@ -10,6 +10,37 @@ Release Notes
    :depth: 1
    :class: twocols
 
+|compass| 1.37.0
+----------------
+
+*Released May 25, 2023*
+
+New Features:
+
+- Enable "Modern Dark Mode" by default
```

Comment:
Is "modern dark mode" the same as the regular dark mode offered by Compass? Can we just say "Enable dark mode by default" ?